packages/workspace-fusion/src/lib/integrations/data-source/types/index.ts

```diff
@@ -5,12 +5,27 @@ export type FetchData<TData> = (signal?: AbortSignal) => TData[] | Promise<TData
 export type { DataSourceConfig };
 
 type DataSourceConfig<TData> = {
-	/** Used to invalidate query */
+	/** Used to control query invalidation.
+	 *
+	 * Defaults to ["Workspace"]
+	 *
+	 * ```TS
+	 * 	const contextId = useContext();
+	 * <Workspace
+	 * {...}
+	 * //Will refetch data when contextId changes
+	 * dataSource={{queryKey: ["Workspace", contextId ]}}
+	 * />
+	 * ```
+	 */
```

Comment:
```suggestion
	/** Used to control query invalidation.
	 *
	 * Defaults to ["Workspace"]
	 * @example
	 * ```TSX
	 * 	const contextId = useContext();
	 * <Workspace
	 * {...}
	 * //Will refetch data when contextId changes
	 * dataSource={{queryKey: ["Workspace", contextId ]}}
	 * />
	 * ```
	 */
```